sdk/storage/Azure.Storage.Common/src/Shared/UriExtensions.cs

```diff
@@ -86,19 +87,11 @@ public static Uri AppendToPath(this Uri uri, string segment)
         /// <returns>Account name or null if not able to be parsed.</returns>
         public static string GetAccountNameFromDomain(string host, string serviceSubDomain)
```

Comment:
Did the changes in this method break anything?

Reply:
I reran all the tests, and everything looks good because the method just accepts more for storage account name, instead of specifically checking whats before the first period, and then checking if the second value after that first period is the serviceSubDomain is what we expect.